Maui.RevenueCat.InAppBilling/Platforms/iOS/Extensions/StoreExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Maui.RevenueCat.InAppBilling.Enums;
+using Maui.RevenueCat.iOS;
+
+namespace Maui.RevenueCat.InAppBilling.Platforms.iOS.Extensions;
+internal static class StoreExtensions
+{
+    internal static StoreId ToStoreId(this RCStore store)
+    {
+        switch (store)
```

Comment:
and here I am responsible for the `StoreId` type -_- 
- will be discussed at the beginning of the PR